mappings/net/minecraft/level/biome/Biome.mapping

```diff
@@ -22,20 +24,25 @@ CLASS kd net/minecraft/level/biome/Biome
 	FIELD w precipitates Z
 	FIELD x biomes [Lkd;
 	METHOD a createBiomeArray ()V
+	METHOD a getBiome (DD)Lkd;
+		ARG 0 x
+		ARG 2 z
 	METHOD a getSkyColour (F)I
 		ARG 1 temperature
-	METHOD a getBiome (FF)Lkd;
-		ARG 0 x
-		ARG 1 z
-	METHOD a setColour_2 (I)Lkd;
+	METHOD a getClimateBiome (FF)Lkd;
+		ARG 0 temperature
+		ARG 1 rainfall
+	METHOD a foliageColour (I)Lkd;
```

Comment:
As to which is grass and which is foliage: I made a guess based on the rainforest colours

Reply:
get prefix on the foliageColour method too?